CMakeLists.txt

```diff
@@ -191,7 +191,7 @@ endif()
 
 # Release flags
 set(RELEASE_CXX_FLAGS
-  -O3 -DNDEBUG
```

Comment:
So now it works? Great!

We probably have fixed some issue in our code.

Reply:
Yes, it works now :)